index.html

```diff
@@ -48,198 +61,254 @@ <h2 class="describe">I am a Software Developer</h2>
+        </li>
+        <li>
+          <a class="icon2" href="#"><img src="images/bootstrap1.png" alt="" /></a>
+        </li>
+        <li>
+          <a class="icon2" href="#"><img src="images/ruby1.png" alt="" /></a>
+        </li>
+      </ul>
 
-  <section class="About-me">
-    <h4 class="about">About me</h4>
-    <p class="about-words">
-      Hello I’m a software developer! I can help you build a product , feature
-      or website Look through some of my work and experience! If you like what
-      you see and have a project you need coded, don’t hestiate to contact me.
-    </p>
+      <button class="button2">See Project</button>
+    </section>
 
-    <li class="resume-button">
-      <a href="#"><img src="images/resume.png" alt="" /></a>
-    </li>
+    <section class="Professional">
+      <h5 class="heading5">Profesional Art Printing Data</h5>
+      <p class="art-words">
+        A daily selection of privately personalized reads; no accounts or sign-ups
+        required. has been the industry's standard
+      </p>
+      <a class="bgimage2"></a>
+      <ul class="f-item">
+        <li>
+          <a class="icon2" href="#"><img src="images/html1.png" alt="" /></a>
+        </li>
+        <li>
+          <a class="icon2" href="#"><img src="images/bootstrap1.png" alt="" /></a>
+        </li>
+        <li>
+          <a class="icon2" href="#"><img src="images/ruby1.png" alt="" /></a>
+        </li>
+      </ul>
+
+      <button class="button2">See Project</button>
+    </section>
+
+    <section class="Professional">
+      <h5 class="heading5">Profesional Art Printing Data</h5>
+      <p class="art-words">
+        A daily selection of privately personalized reads; no accounts or sign-ups
+        required. has been the industry's standard
+      </p>
+      <a class="bgimage2"></a>
+      <ul class="f-item">
+        <li>
+          <a class="icon2" href="#"><img src="images/html1.png" alt="" /></a>
+        </li>
+        <li>
+          <a class="icon2" href="#"><img src="images/bootstrap1.png" alt="" /></a>
+        </li>
+        <li>
+          <a class="icon2" href="#"><img src="images/ruby1.png" alt="" /></a>
+        </li>
+      </ul>
+
+      <button class="button2">See Project</button>
+    </section>
+  </div>
+
+  <section class="About-me">
+    <div class="about-text">
+      <div class="about-column">
+        <h4 class="about">About me</h4>
+        <p class="about-words">
+          Hello I’m a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
 
+        <li class="resume-button">
+          <a href="#"><img src="images/resume.png" alt="" /></a>
+        </li>
+      </div>
+      <a class="bgimage3 desktop-only"></a>
+    </div>
     <hr class="about-break" />
   </section>
```

Comment:
- [ ] I think the background image here is not getting applied correctly. Kindly increase the size to match the Figma design. 
- [ ] Please use the correct `font-family` for the title `About Me`. I think it'll get fixed when you add the required fonts from [Google Font](https://fonts.google.com/). 💯️

**Your Design**
![Screenshot from 2022-12-16 10-06-28](https://user-images.githubusercontent.com/17243325/208042424-45661ec5-85dc-4b39-b6c0-3f5d34237681.png)

**Figma Design**
![Screenshot from 2022-12-16 10-06-04](https://user-images.githubusercontent.com/17243325/208042503-f56800f1-c09f-47c8-b4af-6837e143990e.png)